README.md

```diff
@@ -75,3 +75,29 @@ Most important performance modifications include:
 * avoiding expensive calls of `model.frame()` and `model.matrix` and using the design matrix and response vector instead of a data.frame and a formula
 * using `lm.fit()` instead of `lm()` when possible
 
+## Contribution guidelines:
+We always welcome contributions to the package.
+
+### When you are familiar with GitHub:
+We work with [GitHub Flow](https://guides.github.com/introduction/flow/) branching model.
+
+Key steps:
+1. Create a GitHub issue in this repository with description of the work that you plan to do.
+2. Assign yourself to the GitHub issue you are working on, to inform other developers that you are working on it.
+3. Create your own working branch (or fork) based on the `dev` branch.
+4. Make your changes in that branch or fork.
+5. Commit your changes to your working branch/fork as long as you are not finished with your development.
+6. Make sure the release notes in NEWS.Rd are updated.
+7. Add your name to the contributors list in the DESCRIPTION file.
+8. Run the tests and checks as CRAN, make sure they pass.
```

Comment:
This should be handled by CI and enforced by GitHub anyway, no need to state it.

Reply:
It's a general statement that doesn't hurt IMO.